networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/AttestationTopicSubscriber.java

```diff
@@ -19,42 +19,83 @@
 
 import com.google.common.primitives.UnsignedLong;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Set;
+import tech.pegasys.teku.datastructures.validator.SubnetSubscription;
 import tech.pegasys.teku.networking.eth2.Eth2Network;
 import tech.pegasys.teku.util.time.channels.SlotEventsChannel;
 
 public class AttestationTopicSubscriber implements SlotEventsChannel {
-  private final Map<Integer, UnsignedLong> unsubscriptionSlotBySubnetId = new HashMap<>();
+  private final Map<Integer, UnsignedLong> allSubscriptions = new HashMap<>();
```

Comment:
I think the new name is more confusing - you have to read through the code to figure out what this map represents.
```suggestion
  private final Map<Integer, UnsignedLong> subnetIdToUnsubscribeSlot = new HashMap<>();
```

Reply:
Done.